api/src/test/java/org/openmrs/module/eptsreports/reporting/intergrated/library/cohorts/QualityImprovement2020CohortQueriesTest.java

```diff
@@ -100,24 +100,6 @@ public void getMQ7patientsShouldPass() throws EvaluationException {
     assertNotNull(evaluatedCohort.getMemberIds());
   }
```

Comment:
why did you remove the test methods

Reply:
Edgencio said  that I should remove the methods instead of commenting so the code could be built by the others

Reply:
but these methods were already there  before your implementation